test/unit/cleaner_test.rb

```diff
@@ -18,6 +19,8 @@ class TasksTest < ActiveSupport::TestCase
                          end,
                          FactoryBot.create(:dynflow_task, :product_create_task)]
       cleaner.expects(:tasks_to_csv)
+      # Hack to make the tests pass due to ActiveRecord shenanigans
+      cleaner.expects(:delete_orphaned_dynflow_tasks)
```

Comment:
`ActiveRecord` doesn't actually save things to the db in tests, so the tests fail when accessing AR data using sequel.